tiledb/sm/array_schema/array_schema.h

```diff
@@ -43,11 +43,11 @@
 #include "tiledb/sm/filter/filter_pipeline.h"
 #include "tiledb/sm/misc/constants.h"
 #include "tiledb/sm/misc/hilbert.h"
+#include "tiledb/sm/storage_manager/context_resources.h"
```

Comment:
It doesn't look like this needs to be here, since `store_array_schema` is not a member function in this new revision.